.github/workflows/autobump.yml

```
@@ -0,0 +1,18 @@
+name: Bump the Homebrew cask on release
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  bump-casks:
+    runs-on: macos-latest
+    steps:
+      - uses: macauley/action-homebrew-bump-cask@v1
```

権限を与えてかつ個人のActionsなので、念のためにバージョン固定できるとちょっと安心かもと思いました！
```suggestion
      - uses: macauley/action-homebrew-bump-cask@v1.0.0
```

---

確かにそうですね、ありがとうございます！